rllib/utils/tf_ops.py

```diff
@@ -114,3 +114,20 @@ def scope_vars(scope, trainable_only=False):
         tf.GraphKeys.TRAINABLE_VARIABLES
         if trainable_only else tf.GraphKeys.VARIABLES,
         scope=scope if isinstance(scope, str) else scope.name)
+
+
+def get_shape(x, unknown_value=-1):
```

Comment:
I think we shouldn't add thin wrapper methods like this, since the indirection probably decreases the readability more than the abstraction increases it.

Reply:
Done.